src/algebra/category/Module/filtered_colimits.lean

```diff
@@ -87,6 +87,7 @@ instance colimit_module : module R M :=
 { one_smul := λ x, begin
     apply quot.induction_on x, clear x, intro x, cases x with j x,
     erw [colimit_smul_mk_eq F 1 ⟨j, x⟩, one_smul],
+    dsimp,
```

Comment:
Can you work around the timeout in this declaration by building the `distrib_mul_action_with_zero` or `mul_action` instance first?